roadmaps/2022.md

```diff
@@ -91,33 +91,73 @@ The core team has two main goals for this quarter: to improve the product and al
-- [ ] Keep improving indexation speed/crash
-  - [ ] by gathering user feedback and use-cases and creating benchmarks according to them: we'll ensure we succeeded or not the improvement.
-  - [ ] by keep investigating and trying new technical solutions on our side.
+- [ ] Finish internship work on Jayson. Integrate it to one or two routes on the Meilisearch side. -> We don't know yet if we will prioritize it in Q4 2022 due to the huge breaking it can lead.
+- [X] Work with the devrel team to promote the contributions we need on the tokenizer side -> done during Hacktoberfest, so Q4 2022.
+- [X] Keep improving indexation speed/crash
+  - [ ] by gathering user feedback and use-cases and creating benchmarks according to them: we'll ensure we succeeded or not the improvement -> We are working on creating use-case benchmarks in Q4 2022.
+  - [X] by keep investigating and trying new technical solutions on our side -> we made a lot of changes described in [this issue](https://github.com/meilisearch/meilisearch/issues/2132).
+
+## Q4 2022
+
+This quarter is the last one we have to prepare v1 in good condition. This way, we have prioritized tasks that would help the team deliver Meilisearch v1: a stable and impactful version of Meilisearch.
+It also means we will de-prioritize implementations that could impact in a bad way the critical features of Meilisearch, like improving the indexing speed in a way that would risk making the indexing fail.
+
+Q4 is also the quarter where the Hacktoberfest is happening and this will lead us to spend time on it. We created a lot if issues during Q3, especially regarding the language support, and we hope attract the community and make them contribute in October.
+
+### v1 work
+
+- [ ] Be production ready for v1
+  - [ ] Release the missing production-ready features: index swap, cancel/delete of tasks, and other missing features noticed by the product team.
+  - [ ] We know Meilisearch can have bugs/crashes impacting the stability of the search engine in production. In the best world, the RAM consumption should be appropriate when indexing and searching. It means our priority during this quarter is to identify the different weaknesses of Meilisearch and make decision about them: being transparent with the users is the most important part to achieve if the problem cannot be solved by any implementation before v1.
+  - [ ] Define more precisely what kind of distribution we support. Transparency is the key to this point. Then, we need to ensure we have tests/CIs appropriate to them.
+  For example, what about the Debian 10 distribution? And what about Windows and the prioritization we should give to it? Indeed, all the features of Meilisearch do not work with Windows so far, like dumps. It's important to provide transparency about it.
+- [ ] Define our versioning policy process post v1
+- [ ] Define our beta/deprecated process post v1
```

Comment:
I had good feedback from founders running an open source company at a letter stage than us. I can help with this.